<ui version="4.0" >
 <class>MicroscribeLoggerForm</class>
 <widget class="QMainWindow" name="MicroscribeLoggerForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>186</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Microscribe Logger</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>21</y>
     <width>186</width>
     <height>366</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" >
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <widget class="QGroupBox" name="groupBox" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title" >
           <string>Calibration</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2" >
           <item>
            <widget class="QPushButton" name="btnCalibrate" >
             <property name="text" >
              <string>Calibrate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSetOrigin" >
             <property name="text" >
              <string>Set Origin</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSetPlusX" >
             <property name="text" >
              <string>Set +X</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSetPlusY" >
             <property name="text" >
              <string>Set +Y</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="Experiment" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title" >
           <string>Experiment</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3" >
           <item>
            <widget class="QPushButton" name="btnStartExperiment" >
             <property name="text" >
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnStopExperiment" >
             <property name="text" >
              <string>Stop</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2" >
          <property name="title" >
           <string>Arm</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4" >
           <item>
            <layout class="QGridLayout" name="gridLayout" >
             <item row="0" column="0" >
              <widget class="QLabel" name="label" >
               <property name="text" >
                <string>tx</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" >
              <widget class="QLabel" name="label_tx" >
               <property name="minimumSize" >
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="frameShape" >
                <enum>QFrame::Panel</enum>
               </property>
               <property name="frameShadow" >
                <enum>QFrame::Sunken</enum>
               </property>
               <property name="text" >
                <string>-</string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="2" >
              <widget class="QLabel" name="label_7" >
               <property name="text" >
                <string>mm</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" >
              <widget class="QLabel" name="label_2" >
               <property name="text" >
                <string>ty</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" >
              <widget class="QLabel" name="label_ty" >
               <property name="minimumSize" >
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="frameShape" >
                <enum>QFrame::Panel</enum>
               </property>
               <property name="frameShadow" >
                <enum>QFrame::Sunken</enum>
               </property>
               <property name="text" >
                <string>-</string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="2" >
              <widget class="QLabel" name="label_8" >
               <property name="text" >
                <string>mm</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" >
              <widget class="QLabel" name="label_3" >
               <property name="text" >
                <string>tz</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1" >
              <widget class="QLabel" name="label_tz" >
               <property name="minimumSize" >
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="frameShape" >
                <enum>QFrame::Panel</enum>
               </property>
               <property name="frameShadow" >
                <enum>QFrame::Sunken</enum>
               </property>
               <property name="text" >
                <string>-</string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="2" >
              <widget class="QLabel" name="label_9" >
               <property name="text" >
                <string>mm</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>17</width>
            <height>394</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>186</width>
     <height>21</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>Microscribe Logger</string>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>387</y>
     <width>186</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <action name="actionNew" >
   <property name="text" >
    <string>New</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
